### DISCUSSION TITLE
Feature request: active-active alertmanager cross data centers located in different cities

### DISCUSSION THREAD

**Discussion by jicahoo** [prometheus/alertmanager#2902] [Ideas]
Hi guys,

Hope you are doing well.

In my company, we have two prometheus-alertmanager instances at different data centers. We want to make them active-active:

- Two alertmanagers active at the same time. But they won't send the dup alerts.
- Silence alert manager in one site,  the alert will be also silenced in the other site.
- When one alert manager is down at one site, the alertmanager at another site can still work to send out alert.

It seems that existing HA feature can support above requirements. However, per my understanding, HA feature can only work in LAN environment.   
So is it possible for alertmanager to support these functions cross different  data centers?
It seems that HA feature requires UDP port open, so I think it can only work at LAN environment.  

Thank you.
Best,
Jichao Zhang

**Comment by TheMeier** [comment]
@jicahoo 
Alertmanager HA works just fine across network boundaries. UDP instead of TCP is not a limiting factor.
